kratos/processes/levelset_convection_process.h

```diff
@@ -153,19 +153,19 @@ class LevelSetConvectionProcess
             CalculateNormDxFlag);
 
         mpSolvingStrategy->SetEchoLevel(0);
-        
+
         rBaseModelPart.GetProcessInfo().SetValue(CROSS_WIND_STABILIZATION_FACTOR, cross_wind_stabilization_factor);
-        
+
         //TODO: check flag DO_EXPENSIVE_CHECKS
         mpSolvingStrategy->Check();
 
         KRATOS_CATCH("")
     }
 
     /// Destructor.
-    ~LevelSetConvectionProcess() override 
+    ~LevelSetConvectionProcess() override
     {
-        mrBaseModelPart.GetModel().DeleteModelPart("DistanceConvectionPart");
+        // mrBaseModelPart.GetModel().DeleteModelPart("DistanceConvectionPart");
```

Comment:
this file is used a lot,  and is "in production". Could u comment on what is the change you propose? i am also cc-ing @pablobecker since he is one of the users